conans/client/cmake.py

```diff
@@ -156,3 +165,48 @@ def runtime(self):
         if runtime:
             return "-DCONAN_LINK_RUNTIME=/%s" % runtime
         return ""
+
+    def configure(self, conan_file, args=None, vars=None, source_dir=None, build_dir=None):
```

Comment:
I am merging another PR here, and realized that this is using the built-in "vars", which is not a good idea. I am using "defs" insteads, I think it might be a better name as they are preprocessor defines.